TESTS/netsocket/udp/udpsocket_echotest_burst.cpp

```diff
@@ -168,7 +168,12 @@ void UDPSOCKET_ECHOTEST_BURST_NONBLOCK()
     int bt_total = 0;
     for (int i = 0; i < BURST_CNT; i++) {
         for (int x = 0; x < BURST_PKTS; x++) {
-            TEST_ASSERT_EQUAL(tx_buffers[x].len, sock.sendto(udp_addr, tx_buffers[x].payload, tx_buffers[x].len));
+            nsapi_size_or_error_t sent = sock.sendto(udp_addr, tx_buffers[x].payload, tx_buffers[x].len);
+            if (sent != NSAPI_ERROR_WOULD_BLOCK) {
+                TEST_ASSERT_EQUAL(tx_buffers[x].len, sent);
+            } else {
+                x--;
```

Comment:
There is no delay here.. maybe it needs to wait for SIGIO to happen before retrying..

Reply:
Yes, but as retry logic is not yet implemented in this test case, so I would like to leave actual implementation for netsocket developers. :)